<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="mattibharath" failures="0" tests="2" name="com.comcast.crm.Contact_Test.CreatingAContact_Test" time="17.209" errors="1" timestamp="2024-06-04T12:48:30 IST" skipped="0">
  <testcase classname="com.comcast.crm.Contact_Test.CreatingAContact_Test" name="closingTheDbConnection" time="3.439">
    <error message="Cannot invoke &quot;java.sql.Connection.close()&quot; because &quot;this.conn&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.sql.Connection.close()" because "this.conn" is null
at com.generic.databaseutility.DataBaseUtility.CloseDataBaseConnection(DataBaseUtility.java:62)
at com.comcast.crm.BaseclassUtility.Baseclass_Test.closingTheDbConnection(Baseclass_Test.java:122)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:344)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- closingTheDbConnection -->
  <system-out/>
  <testcase classname="com.comcast.crm.Contact_Test.CreatingAContact_Test" name="creatingcontact" time="13.770"/>
  <system-out/>
</testsuite> <!-- com.comcast.crm.Contact_Test.CreatingAContact_Test -->
